static/app/components/dropdownMenuControlV2.tsx

```diff
@@ -0,0 +1,193 @@
+import {
+  ElementType,
+  HTMLAttributes,
+  MouseEvent,
+  ReactNode,
```

Comment:
You don't need to import these types, just use `React.ReactNode` etc etc

Reply:
It's all in the global namespace

Reply:
Oh wasn't aware of that! Will remove them